ci-operator/jobs/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,61 @@
 presubmits:
   openshift/cluster-image-registry-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/cluster-image-registry-operator-e2e-aws
+    decorate: true
+    name: pull-ci-openshift-cluster-image-registry-operator-master-cluster-image-registry-operator-e2e-aws
+    rerun_command: /test cluster-image-registry-operator-e2e-aws
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/cluster-image-registry-operator-e2e-aws-cluster-profile
+        - --target=cluster-image-registry-operator-e2e-aws
+        - --template=/usr/local/cluster-image-registry-operator-e2e-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-cluster-image-registry-operator-master.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: cluster-image-registry-operator-e2e-aws
+        - name: TEST_COMMAND
+          value: go test ./test/e2e
+        image: ci-operator:latest
```

Comment:
@smarterclayton @stevekuznetsov how do i get this value to be my own image, since the generator keeps overwriting it?

Reply:
You mean use a custom ci-operator image?